client/reader/list-stream/missing.jsx

```diff
@@ -40,6 +41,7 @@ class ListMissing extends React.Component {
 
 		return (
 			<div>
+				<DocumentHead title={ this.props.translate( 'List not found' ) } />
 				<QueryReaderList owner={ this.props.owner } slug={ this.props.slug } />
```

Comment:
Do we still need to query for the list if we're already rendering `missing.jsx`?

Reply:
Nope, fixed! :)